README.md

```diff
@@ -46,10 +46,12 @@ OpenTelemetry is the merging of OpenCensus and OpenTracing into a single project
 This project contains the following top level components:
 
 * [OpenTelemetry API](api/):
-  * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, `Meter`, and `Baggage`
-  * [context api](context/src/main/java/io/opentelemetry/context/) The OpenTelemetry Context implementation.
-* [extensions](extensions/) define additional API extensions, which are not part of the core API.
+  * [stable apis](api/all) including `Tracer`, `Span`, `SpanContext`, `Meter`, and `Baggage`.
+  * [context api](context/) The OpenTelemetry Context implementation.
+  * [experimental apis](api/all) including `Events`.
+* [extensions](extensions/) define additional API extensions not part of the core API, including propagators and experimental extension APIs.
```

Comment:
maybe `opentelemetry-extension-incubator` -> `opentelemetry-api-incubator`?

since it's not really an incubator for extensions

(also https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation-api-incubator)

Reply:
Tracking issue here: https://github.com/open-telemetry/opentelemetry-java/issues/6121